debian/changelog

```diff
@@ -19,11 +19,37 @@ cloud-init (23.4.1-0ubuntu1~20.04.3) UNRELEASED; urgency=medium
     - d/p/retain-netplan-world-readable.patch
     - d/p/status-do-not-remove-duplicated-data.patch
     - d/p/status-retain-recoverable-error-exit-code.patch
-    - d/p/keep-dhclient-as-priority-client.patch
-    - d/p/revert-551f560d-cloud-config-after-snap-seeding.patch
   * Upstream snapshot based on upstream/main at d8b6ac2b.
```

Comment:
Let's also prune duplicate Upstream snapshot comments from d/changelog on this yet to be released entry